recirq/otoc/loschmidt/tilted_square_lattice/analysis.py

```diff
@@ -136,11 +141,15 @@ def fit_vs_macrocycle_depth(df):
 
     Returns:
         fitted_df: A new dataframe containing fit parameters.
-        exp_ansatz_vs_macrocycle_depth: The function used for the fit.
+        exp_ansatz_vs_macrocycle_depth: The function used for the fit. This is
+            a * f^depth. The depth in this expression is the number of macrocycles
+            multiplied by four (to give the number of cycles) and multiplied by two
+            (to account for the inversion in the echo).
     """
 
     def exp_ansatz_vs_macrocycle_depth(macrocycle_depths, a, f):
-        return a * np.exp(np.log(f) * 4 * 2 * macrocycle_depths)
+        return a * np.exp(
```

Comment:
nit: Wondering if it makes sense to have a negative in the exponent. I know this is equivalent since `f` can just be < 1, but perhaps having a negative gives better intuition around the shape of the graph (that it's a decreasing exponential)

Reply:
The intuition is f^d. We are much more comfortable talking about the per-layer fidelity than an exponential decay constant. happy to talk more if I'm not understanding you correctly.